src/app/result/result.component.css

```diff
@@ -45,31 +169,40 @@ hr {
   transition: opacity 0.5s ease-in-out 0.2s;
 }
 
-.purchase_btn {
-  margin-top: 17rem;
-  margin-left: 3rem;
-  position: absolute;
 
-  visibility: hidden;
-  z-index: 20;
+.card_desc_italicize {
+  font-style: italic;
+}
 
+.purchase_btn {
   width: 11rem;
   height: 4rem;
   background-color: orange;
   color: whitesmoke;
   border-radius: 3px;
   border: yellow;
+  font-size: small;
 
   transition: opacity 1s;
 }
 
 .email_holder {
-  z-index: 30;
+  /*z-index: 30;*/
```

Comment:
better to just remove the whole thing unless you have a good reason to keep it as a comment. If so, write why you're keeping it